src/vaadin-date-picker.html

```diff
@@ -57,12 +57,12 @@
         disabled="[[disabled]]"
         readonly="[[readonly]]"
         error-message="[[errorMessage]]"
+        clear-button-visible="[[clearButtonVisible]]"
         aria-label$="[[label]]"
         part="text-field"
         theme$="[[theme]]"
       >
       <slot name="prefix" slot="prefix"></slot>
-      <div part="clear-button" slot="suffix" on-touchend="_clearTouchend" on-click="_clear" role="button" aria-label$="[[i18n.clear]]"></div>
```

Comment:
https://github.com/vaadin/vaadin-date-picker/blob/33cfb8457779177c6e7d5c57143f8002eb350b7d/src/vaadin-date-picker.html#L39

This CSS block should be removed since there is no more `part="clear-button"` in this template.

Reply:
There's also this clear button:  
https://github.com/vaadin/vaadin-date-picker/blob/33cfb8457779177c6e7d5c57143f8002eb350b7d/src/vaadin-date-picker-overlay-content.html#L165

which I think should be removed. It looks like that one is never shown. It also has some associated CSS which could then be removed (in the same template and also in `vaadin-date-picker-overlay-content-styles.html`).

It also looks like the `_clear()` methods in both `vaadin-date-picker.html` and also `vaadin-date-picker-overlay-content.html` are not used anywhere so those could probably be removed?

Reply:
> which I think should be removed. It looks like that one is never shown.

This clear button is shown in the overlay in Material theme on mobile.

Reply:
BTW here you can try a material date-picker: https://cdn.vaadin.com/vaadin-material-styles/1.2.2/demo/date-pickers.html

Reply:
Ok, I didn't notice it's only shown with Material theme. I just looked that with Lumo it was not shown in desktop or mobile. I think it's a bit weird that a control in a specific view is only shown in one theme. I wonder what's the reason it's not shown with Lumo. But I guess this is fine now. Thanks.